var/spack/repos/jcsda-emc-bundles/packages/jedi-neptune-env/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class JediNeptuneEnv(BundlePackage):
+    """Development environment for neptune-bundle"""
+
+    # Fake URL
+    homepage = "https://github.com/JCSDA-internal/neptune-bundle"
+    git = "https://github.com/JCSDA-internal/neptune-bundle.git"
+
+    maintainers("climbfuji", "areineke")
+
+    version("1.0.0")
+
+    depends_on("jedi-base-env", type="run")
+
+    depends_on("libyaml", type="run")
+    depends_on("p4est", type="run")
+    depends_on("w3emc", type="run")
+    depends_on("esmf~debug", type="run")
```

Comment:
I just noticed in jedi-ufs-env, `~debug` is specified for esmf and mapl. Is that needed, as in, the applications being supported can't/won't work with `+debug`?

Reply:
No, actually not. This is a remainder of earlier times when we had +debug and ~debug in the ufs-weather-model env, and we wanted to make sure that these packages were using the correct release builds. I should remove those.